apps/ex_wire/mix.exs

```diff
@@ -34,6 +34,7 @@ defmodule ExWire.Mixfile do
     [
       {:ex_doc, "~> 0.16", only: :dev, runtime: false},
       {:ex_rlp, "~> 0.3.1"},
+      {:cli, in_umbrella: true},
```

Comment:
though CLI is technically not an application. it doesn't have an application module.